packages/js/src/core/user-info.test.ts

```diff
@@ -0,0 +1,11 @@
+import { fetchUserInfo, UserInfoResponse } from './user-info';
+
+describe('fetchUserInfo', () => {
+  test('should return UserInfoResponse', async () => {
+    const userInfoResponse: UserInfoResponse = { sub: 'foo' };
+    const requester = jest.fn().mockResolvedValue(userInfoResponse);
+    await expect(
+      fetchUserInfo('https://example.com/oidc/me', 'access_token_value', requester)
+    ).resolves.toEqual(userInfoResponse);
```

Comment:
`toMatchObject`?

Reply:
I'm ok with both. 
Name of `toMatchObject` seems to be more precise.
Updated, PTAL.